src/tests/miscellaneous/mock-adb/README.md

```diff
@@ -3,39 +3,8 @@ Copyright (c) Microsoft Corporation. All rights reserved.
 Licensed under the MIT License.
 -->
 
-## mock-adb
+# mock-adb test integration
 
-This directory contains the implementation for a `mock-adb` (Android Debug Bridge) executable. The unified product leverages `adb` to setup a test Android device. We use `mock-adb` in two ways:
-* during end-to-end tests that exercise the Android setup process
-* during development if you prefer to use a mock device/emulator
+This directory contains helpers to use mock-adb from within unified test cases.
```

Comment:
Suggestion: rename this directory to line up with this description (`src/tests/miscellaneous/mock-adb-helpers` for example). That way we have a more descriptive name and avoid having two `mock-adb` directories in different locations

Reply:
Done! I went with `setup-mock-adb` instead of `mock-adb-helpers`